core/src/main/java/org/apache/james/core/quota/QuotaLimit.java

```diff
@@ -20,25 +20,26 @@
 package org.apache.james.core.quota;
 
 import java.util.Objects;
+import java.util.Optional;
 
 public class QuotaLimit {
 
-    public static QuotaLimit of(QuotaComponent component, QuotaScope scope, String identifier, QuotaType quotaType, long limit) {
+    public static QuotaLimit of(QuotaComponent component, QuotaScope scope, String identifier, QuotaType quotaType, Long limit) {
```

Comment:
Missed that the first time but is that static method useful here as well?

Reply:
I saw this method in other places of james. I think we prefer this way to create objects :))

Reply:
Like where?

Reply:
org/apache/james/jmap/api/filtering/Rule.java - class Condition - line 186

Reply:
Missed your constructors were private. My bad